scheduling/quartz/src/main/java/io/quarkus/ts/scheduling/quartz/failover/AnnotationScheduledJob.java

```diff
@@ -17,10 +21,24 @@ public class AnnotationScheduledJob {
     @Inject
     ExecutionService service;
 
+    @Inject
+    InternalApplicationNotRunning appNotRunning;
+
     @Transactional
-    @Scheduled(cron = "0/1 * * * * ?", identity = "my-unique-task")
+    @Scheduled(cron = "0/1 * * * * ?", identity = "my-unique-task", concurrentExecution = SKIP)
     void increment() {
-        service.addExecution(ownerName);
+        if (appNotRunning.isAppRunning()) {
+            service.addExecution(ownerName);
+        }
+    }
+
+    // necessary as 'ApplicationNotRunning' is not registered with Quartz extension
+    @Singleton
+    public static class InternalApplicationNotRunning extends ApplicationNotRunning {
+
+        private boolean isAppRunning() {
```

Comment:
Do we control the name of this method? If so, can we change it to something like `check` or `verify`? `appNotRunning.isAppRunning()` is a bit of a mind-twister

Reply:
renamed to `verifyAppIsRunning`

Reply:
okay @fedinskiy I didn't read it together :-D Verify it is.

Reply:
No, it still doesn't make sense. Will try again.

Reply:
@fedinskiy I couldn't make naming work with appNotRunning.verify == app is running, so I introduced boolean variable instead.